recirq/otoc/otoc_example.py

```diff
@@ -77,33 +90,28 @@ def main():
     for i, circuits_i in enumerate(circuit_list):
         results_i = np.zeros((num_qubits, len(cycles)))
         for c, circuits_ic in enumerate(circuits_i):
-            print('Measuring circuit instance {}, cycle {}...'.format(i, c))
+            print("Measuring circuit instance {}, cycle {}...".format(i, c))
             stats = int(2000 + 10000 * (c / max(cycles)) ** 3)
-            sweep_params = [{} for _ in range(len(circuits_ic))]
-            job = engine.run_batch(
-                programs=circuits_ic,
-                params_list=sweep_params,
-                repetitions=stats,
-                processor_ids=[processor_name],
-                gate_set=cirq.google.FSIM_GATESET)
+            params = [{} for _ in range(len(circuits_ic))]
+            job = sampler.run_batch(programs=circuits_ic, params_list=params, repetitions=stats)
             for d in range(num_qubits):
-                p = np.mean(job.results()[4 * d].measurements['z'])
-                p -= np.mean(job.results()[4 * d + 1].measurements['z'])
-                p -= np.mean(job.results()[4 * d + 2].measurements['z'])
-                p += np.mean(job.results()[4 * d + 3].measurements['z'])
+                p = np.mean(job[4 * d][0].measurements["z"])
+                p -= np.mean(job[4 * d + 1][0].measurements["z"])
+                p -= np.mean(job[4 * d + 2][0].measurements["z"])
+                p += np.mean(job[4 * d + 3][0].measurements["z"])
```

Comment:
These `[0]`. What are they? Is this from the "params" argument? If you omit `params_list` can you get rid of this `[0]`?

Reply:
I don't know what they are. I have to specify them even if I omit params_list with Cirq v0.10. In the `run_batch method`, the return type is specified to be `List[List['cirq.Result']]`, so it is definitely a nested list for some reason.

Reply:
🤷‍♂️